src/components/shared/ResultList.js

```diff
@@ -38,24 +40,41 @@ export default class ResultList extends Component<Props> {
       key: `${item.id}${item.value}${index}`,
       ref: String(index),
       title: item.value,
+      "aria-labelledby": `${item.id}-title`,
+      "aria-describedby": `${item.id}-subtitle`,
+      role: "option",
       className: classnames("result-item", {
         selected: index === selected
       })
     };
 
     return (
       <li {...props}>
-        <div className="title">{item.title}</div>
-        <div className="subtitle">{item.subtitle}</div>
+        <div id={`${item.id}-title`} className="title">
+          {item.title}
+        </div>
+        <div id={`${item.id}-subtitle`} className="subtitle">
+          {item.subtitle}
+        </div>
       </li>
     );
   }
 
   render() {
-    const { size, items } = this.props;
+    const { size, items, role, selected } = this.props;
 
     return (
-      <ul className={classnames("result-list", size)}>
+      <ul
+        className={classnames("result-list", size)}
+        role={role}
+        aria-activedescendant={
+          typeof items !== undefined && items.length > 0
+            ? items[selected].id + "-title"
+            : null
+        }
+        aria-expanded={typeof items !== undefined && items.length > 0}
```

Comment:
aria-expanded belongs to the parent of the input field instead of the list. Move it there. 

That parent ("search-field"), should also have role="combobox" and aria-haspopup="listbox" attributes (For more details and a really good example see https://www.w3.org/TR/wai-aria/roles#combobox).  

Because "search-field" element is not an ancestor or the results list, lets add aria-owns="result-list" to it as well.